src/operation.ts

```diff
@@ -326,19 +326,34 @@ export default class Operation {
    * Get an `operationId` for this operation. If one is not present (it's not required by the spec!) a hash of the path
    * and method will be returned instead.
    *
+   * @param shouldCamelCase Generate a JS method-friendly operation ID when an `operationId` isn't present.
    */
-  getOperationId(): string {
+  getOperationId(shouldCamelCase = false): string {
     if ('operationId' in this.schema) {
       return this.schema.operationId;
     }
 
-    const url = this.path
+    const method = this.method.toLowerCase();
+    let operationId = this.path
       .replace(/[^a-zA-Z0-9]/g, '-') // Remove weird characters
       .replace(/^-|-$/g, '') // Don't start or end with -
       .replace(/--+/g, '-') // Remove double --'s
       .toLowerCase();
 
-    return `${this.method.toLowerCase()}_${url}`;
+    if (shouldCamelCase) {
```

Comment:
if I'm reading this right, we're keeping this backwards compatible? Are you going to enable camel case for anything but the api lib?

I ask because we probably don't want to break explorer links.

Reply:
CamelCase work is only for the API library.